src/main/java/mil/dds/anet/beans/search/SubscribableObjectSearchQuery.java

```diff
@@ -0,0 +1,47 @@
+package mil.dds.anet.beans.search;
+
+import io.leangen.graphql.annotations.GraphQLInputField;
+import io.leangen.graphql.annotations.GraphQLQuery;
+import java.util.Objects;
+
+public abstract class SubscribableObjectSearchQuery<T extends ISortBy>
+    extends AbstractSearchQuery<T> {
+
+  @GraphQLQuery
+  @GraphQLInputField
+  private Boolean subscribed;
+
+  public SubscribableObjectSearchQuery(T defaultSortBy) {
+    super(defaultSortBy);
+  }
+
+  public boolean getSubscribed() {
+    return Boolean.TRUE.equals(subscribed);
+  }
+
+  public void setSubscribed(Boolean subscribed) {
+    this.subscribed = subscribed;
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(super.hashCode(), subscribed);
```

Comment:
isn't `Objects.hash(super, subscribed);` sufficient?

Reply:
Actually, yes, but other places already called it like this, so either we should change them all or leave it like this.